hack/build-images.sh

```diff
@@ -138,6 +138,9 @@ image openshift/origin-sti-builder           images/builder/docker/sti-builder
 image openshift/origin-f5-router             images/router/f5
 image openshift/node                         images/node
 
+# images used for tests
+image openshift/test-multicast               images/testing/multicast
```

Comment:
OK, created https://github.com/openshift/test-multicast. I assume the "openshift" account on docker hub needs to set up the build? Who can do that?